frontend/src/styles/TeamStyle.css

```diff
@@ -0,0 +1,29 @@
+.card{
+    text-align: center;
+    width: 350px;
+    padding: 30px;
+    margin-right: 20px;
+    margin-top: 20px;
+    background-color: #A6E2DC;
+    
+    
+}
+.card-title{
+    font-family: Arial, Helvetica, sans-serif;
+    color: black;
+    font-weight: bold;
+    font-size: 150%;
+}
+.card-text{
+    text-align: center;
+    font-family: Arial;
+    font-weight: bold;
+    font-size: 100%;
+}
+.card-image{
+    width: 50%;
+}
+.card-style{
+    background-color: #A6E2DC;
+    
```

Comment:
Consider naming these classes something more unique. Leaving them as is will affect any component that will use the card, card-image, card-title, or any of the others as well.